test/data/gvcf_test_cases/vcf_services.yml

```diff
@@ -1,5 +1,5 @@
 readme: |
-  This is a gvcf testing of the unify_sites service.
+  This is a gvcf testing of the unify_sites service. It involves a case where the unified site is longer than the vcf record. This site is callable assuming that bases not in a vcf record are treated as REF.
 
```

Comment:
This is a slightly strange example, since it is not gvcf, so I'm making the assumption that any regions that are not reported in the vcf will be treated as REF.